linux-anvil-aarch64-cuda/Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG CUDA_VER
 ARG CENTOS_VER
-FROM --platform=linux/arm64 nvidia/cuda:${CUDA_VER}-devel-centos${CENTOS_VER}
+FROM --platform=linux/arm64 nvidia/cuda:${CUDA_VER}-devel-${CENTOS_VER}
```

Comment:
I've changed the meaning of `CENTOS_VER` to be more than an integer here, only for aarch64 and ppc64le (`"centos7"` or `"ubi8"` instead of just `"8"`). This is primarily a short-term fix to see if CI passes. I would like to know if others think this is too hacky, and if so, suggestions for what to do instead (introduce another variable named `IMAGE_BASE`, perhaps?).

Reply:
@jakirkham @kkraus14 @jaimergp CI is passing now, with one caveat that the meaning of `CENTOS_VER` has changed for the aarch64 and ppc64le CUDA images as mentioned above. If you have thoughts on how this should move forward, please let me know.

Reply:
This seems ok. If we wanted to be clearer, maybe we could do `${DISTRO_NAME}${DISTRO_VER}`

Reply:
That sounds good to me! Should that be done in this PR or a follow-up? I would like to merge this and keep the scope constrained to CUDA 11.7, then open a separate PR to refactor that for all images.

Reply:
Given we are changing the meaning of the term in this PR, it seems changing the name would be appropriate in this PR. It could be refactored into an earlier PR that we merge first, but it should get handled in some way.

FWIW I started to do this here ( https://github.com/jakirkham/conda-forge-docker-images/commit/9dfc91f2759af5e3bbe9d628d2443b90937098a3 ), but am being pulled into other things so may not be able to finish it. Feel free to grab that as a starting point.

Reply:
Submitted PR ( https://github.com/conda-forge/docker-images/pull/215 )

Reply:
That's now in. So this can be rebased or merged with that

Reply:
This update is now complete! I'll resolve this thread.